src/Stratis.Bitcoin.Features.Wallet/WalletRPCController.cs

```diff
@@ -29,13 +41,51 @@ public uint256 SendToAddress(BitcoinAddress bitcoinAddress, Money amount)
             return uint256.Zero;
         }
 
+        /// <summary>
+        /// Broadcasts a raw transaction from hex to local node and network.
+        /// </summary>
+        /// <param name="hex">Raw transaction in hex.</param>
+        /// <returns>The transaction hash.</returns>
+        [ActionName("sendrawtransaction")]
+        [ActionDescription("Submits raw transaction (serialized, hex-encoded) to local node and network.")]
+        public async Task<uint256> SendTransactionAsync(string hex)
+        {
+            this.logger.LogTrace("({0}:'{1}')", nameof(hex), hex);
+
+            Transaction transaction = this.fullNode.Network.CreateTransaction(hex);
+            await this.broadcasterManager.BroadcastTransactionAsync(transaction);
+
+            uint256 hash = transaction.GetHash();
+
+            this.logger.LogTrace("(-):'{0}'", hash);
+            return hash;
+        }
+             
+        /// <summary>
+        /// RPC method that gets a new address for receiving payments.
+        /// Uses the first wallet and account.
+        /// </summary>
+        /// <returns>The new address.</returns>
+        [ActionName("getnewaddress")]
+        [ActionDescription("Returns a new wallet address for receiving payments.")]
+        public NewAddressModel GetNewAddress()
+        {
+            this.logger.LogTrace("()");
+
+            HdAddress hdAddress = this.walletManager.GetUnusedAddress(this.GetAccount());
+            string base58Address = hdAddress.Address;
+
+            this.logger.LogTrace("(-):'{0}'", base58Address);
+            return new NewAddressModel(base58Address);
+        }
+
         private WalletAccountReference GetAccount()
         {
             //TODO: Support multi wallet like core by mapping passed RPC credentials to a wallet/account
-            string w = this.WalletManager.GetWalletsNames().FirstOrDefault();
+            string w = this.walletManager.GetWalletsNames().FirstOrDefault();
```

Comment:
`account 0` is the default, but I think that long-standing TODO has a point - ultimately some form of credential needs to be made available to the RPC functionality so that the methods that sign transactions can be implemented. Not saying that has to be in this PR, but it should be borne in mind.

Reply:
We are going to have to have a bigger discussion on wallet RPC methods in general. I need to implement sendtoaddress which requires signing, but I'll create a separate issue for discussion on how best to do this.

Reply:
I created a issue #2266 if you guys have any suggestions on sendtoaddress